### ISSUE TITLE
Create changelog fails on windows

### ISSUE THREAD

**Issue by sudo-suhas** [webpro/release-it#89]
My config:
```json
{
  "verbose": true,
  "buildCommand": "npm run build",
  "requireCleanWorkingDir": true,
  "src": {
    "tagName": "v%s",
    "beforeStartCommand": "npm run check"
  },
  "dist": {
    "baseDir": false
  },
  "github": {
    "release": true
  },
  "npm": {
    "publish": true
  }
}
```

Verbose output:
```
E:\Projects\repos\elastic-builder (master) (elastic-builder@0.2.1)
λ release-it minor
[execute] git config --get remote.origin.url
https://github.com/sudo-suhas/elastic-builder.git
[execute] git show-ref --tags --quiet --verify -- "refs/tags/v0.2.1"
[execute] git log --pretty=format:'* %s (%h)' v0.2.1...HEAD
fatal: ambiguous argument '%s': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
WARNING Probably the current version in package.json is not a known tag in the repository.
ERROR Could not create changelog from latest tag (v0.2.1) to HEAD.
```
Dry run does not detect this issue and only fails when trying to release it.

Setting the `changelogCommand` property fixes it:
```
"changelogCommand": "git log --pretty=format:\"* %s (%h)\" [REV_RANGE]"
```

Would it be possible to detect the OS and change default settings? Or at least could we add a note to the README?

**Comment by webpro** [comment]
Can you verify that your issue is specific to Windows? It is perhaps due to single/double quotes behavior is different in your shell?

**Comment by sudo-suhas** [comment]
The command with single quotes works on git-bash and powershell works but fails on command prompt(and cmder). On the other hand, command with double quotes works on all.

**Comment by sudo-suhas** [comment]
Can't find an exact reference but this might be relevant - https://groups.google.com/forum/#!msg/msysgit/qzKNBrjYUUM/pYSXqpEiEBMJ

**Comment by webpro** [comment]
I've released [v2.7.3](https://github.com/webpro/release-it/releases/tag/2.7.3) which uses double quotes for this. Hope this helps :-)

**Comment by sudo-suhas** [comment]
Thanks!
